### DISCUSSION TITLE
Adaptation of Gladiator mode

### DISCUSSION THREAD

**Discussion by PavanLuca** [redeclipse/base#1531] [Ideas and Feature Requests]
Playing in **Gladiator** mode grenades and mines shouldn't appear otherwise it becomes a **Kaboom** mode.

**Comment by SniperGoth** [comment]
But that's the point, mines/grenades are extras you can pick up.
Gladiator kinda acts like a mini Smash Bros, in a way, but it needs more items.

**Reply by PavanLuca** [reply]
The problem is that since it's **Gladiator** it implies that (even by definition) you only use swords and, at most, kicks. And, telling the truth, players would definitely use explosives on **Gladiator** matches to have the advantage of killing in distance rather than risking the death in a hand-to-hand fight so this would take the match out of the expectation of hand-to-hand fights precisely. This advantage isn't what **Gladiator** matches should encourage. For short, if you want to bomb a map you play **Kaboom** 🤪😆.
